packages/office-ui-fabric-react/src/components/Keytip/examples/Keytip.Basic.Example.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { IKeytipProps, Keytip } from 'office-ui-fabric-react/lib/Keytip';
+import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
+import { convertSequencesToKeytipID } from 'office-ui-fabric-react/lib/Utilities';
+
+export interface IKeytipExampleState {
+  keytipVisible: boolean;
+}
+
+export interface IKeytipMap {
+  [componentKeytipId: string]: IKeytipProps;
+}
+
+export function onKeytipButtonClick(): void {
```

Comment:
Since this uses setState, shouldnt it just be an instance method on KeyTipBasicExample?